megalinter/reporters/AzureCommentReporter.py

```diff
@@ -116,7 +116,7 @@ def produce_report(self):
                     + "See https://learn.microsoft.com/en-us/azure/devops/pipelines/"
                     + "build/variables?view=azure-devops&tabs=yaml"
                 )
-                repository_name = SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI.split("/")[-1]
+                repository_name = SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI.split("/")[-1].replace('%20', ' ')
```

Comment:
Is it only spaces that need to be encoded? Or a general urlencoding should be used?

Reply:
It's more decoding from the URL and spaces should be sufficient based on the ADO imposed restrictions?

https://learn.microsoft.com/en-us/azure/devops/organizations/settings/naming-restrictions?view=azure-devops#azure-repos-git